templates/includes/sidebar.html

```diff
@@ -1,4 +1,15 @@
 <ul class="sidebar-nav">
+  <li>
+    <nav>
+      {% if user.is_authenticated %}
+        <li>User: {{ user.username }}</li>
+        <li><a href="{% url 'logout' %}">Logout</a></li>
+        {% else %}
+        <li><a href="{% url 'login' %}">Login</a></li>
+      {% endif %}
+    <br>
+    </nav>
+  </li>
```

Comment:
The `<li>` element should not directly contain a `<nav>` element. Consider restructuring the HTML to ensure that list items (`<li>`) contain only inline elements or block elements that are semantically appropriate. You might want to wrap the `<nav>` content within a single `<li>` or adjust the structure to maintain semantic HTML.